packages/features/ee/dsync/lib/removeUserFromOrg.ts

```diff
@@ -0,0 +1,11 @@
+import removeMember from "@calcom/features/ee/teams/lib/removeMember";
+
+const removeUserFromOrg = async ({ userId, orgId }: { userId: number; orgId: number }) => {
+  await removeMember({
+    memberId: userId,
+    teamId: orgId,
+    isOrg: true,
+  });
+};
```

Comment:
No need to await in here, we can just return the promise and `await removeUserFromOrg`.
```suggestion
const removeUserFromOrg = async ({ userId, orgId }: { userId: number; orgId: number }) =>
  removeMember({
    memberId: userId,
    teamId: orgId,
    isOrg: true,
  });
```

Reply:
Fixed in #13850